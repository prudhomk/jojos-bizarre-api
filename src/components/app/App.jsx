import React from 'react';
import { Route, Switch } from 'react-router-dom';
import CharacterList from '../characters/CharacterList';
import StandList from '../stands/StandList';
import Home from '../home/Home';

export default function App() {
  return (
    <>
      <h1>Hello World</h1>
      <Switch>
        <Route exact path="/" component={Home}/>
        <Route exact path="/characters" component={CharacterList}/>
        <Route exact path="/stands" component={StandList}/>
      </Switch>
    </>
  );
}
