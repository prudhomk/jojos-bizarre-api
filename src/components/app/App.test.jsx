import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { render, cleanup } from '@testing-library/react';
import App from './App';


describe('Snapshot', () => {
  afterEach(() => cleanup());
  it('renders App', () => {
    const { asFragment } = render(<MemoryRouter><App /></MemoryRouter>);
    expect(asFragment()).toMatchSnapshot();
  });
});



