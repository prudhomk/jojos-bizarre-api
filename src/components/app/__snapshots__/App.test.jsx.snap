// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot renders App 1`] = `
<DocumentFragment>
  <h1>
    Hello World
  </h1>
  <h1>
    Home
  </h1>
</DocumentFragment>
`;
